### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by shruthidasarapu**
# Description

Please provide a short summary explaining what this PR is about.
Homepage

Fixes # https://github.com/HackYourFuture-CPH/rate-my-cv/issues/119

# How to test?
npm run dev

Please provide a short summary how your changes can be tested?

# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by arya56 — COMMENTED** [review]
@shruthidasarapu the app.js file is not updated, due to the problem we had yesterday regarding the header and footer, that's why u don't have header on top, try to pull from Dev, and merge it to this branch and commit and push ,, then we can see header on top of homepage.

File: src/client/containers/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.write-resume {
```
Comment (arya56) [inline src/client/containers/Home/Home.css:1]: @shruthidasarapu @KavyaReddy732 , to get ride of horizontal scrollbar we need ```width:100%``` here: ```.homepage-slogan:nth-of-type(1)```

Comment (arya56) [reply src/client/containers/Home/Home.css:1]: @shruthidasarapu also images from ``` TitleDesc``` would in perfect size if we give them ```width:83%``` & can be added to this class: ```.write-resume img``` (was made by @DiegoRuiz454 )

Comment (arya56) [reply src/client/containers/Home/Home.css:1]: from uploaded cv section we need some adjustments to the font-family and font size

**Comment by pdsorensen** [comment]
Please fix the build step before requesting rewiews. Also, can i have you respond on Alis review?

**Review by CharlesKimMaina — COMMENTED** [review]
Looks great,.,.

File: src/client/containers/Home/Home.js [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+      </div>
+      <div className="instruction">
+        <InstructionsComponent />
+      </div>
+      <div className="write-resume gray-background">
+        <TitleDesc
+          title="Write your Resume"
+          description="Amet minim mollit non deserunt ullamco est sit allque 
+          dolor do amet sint. Velit officia consequat duls enim 
+          velit mollit. Exercitation veniam consequat sunt nostrud amet"
```
Comment (bytewiz) [inline src/client/containers/Home/Home.js:41]: I believe there would be an error I am not sure you can make multiple line strings in the editor like this?

File: src/client/containers/Home/Home.js [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+        />
+        <img src={image1} alt="right" />
+      </div>
+      <div className="write-resume">
+        <img src={image2} alt="left" />
+        <TitleDesc
+          title="Review Resumes"
+          description="Amet minim mollit non deserunt ullamco est sit allque
```
Comment (bytewiz) [inline src/client/containers/Home/Home.js:49]: I believe there would be an error I am not sure you can make multiple line strings in the editor like this?

**Review by bytewiz — CHANGES_REQUESTED** [review]

File: src/client/containers/Home/Home.js (hunk 2 above)
Comment (shruthidasarapu) [reply src/client/containers/Home/Home.js]: I did those changes .

**Review by bytewiz — APPROVED** [review]

**Pull request merged**